cloudcix_primitives/vpnroute_ns.py

```diff
@@ -0,0 +1,275 @@
+        3053: f'Failed to connect to the disabled PodNet from the config file {config_file} for payload remove_existing_routes: ',
+        3054: f'Failed to run remove_existing_routes payload on the disabled PodNet. Payload exited with status ',
+        3055: f'Failed to connect to the disabled PodNet from the config file {config_file} for payload add_route: ',
+        3056: f'Failed to run add_route payload for network %(network)s on the disabled PodNet. Payload exited with status ',
+    }
+
+    # Default config_file if it is None
+    if config_file is None:
+        config_file = '/opt/robot/config.json'
+
+    # Check if networks list is empty
+    if not networks:
+        return False, messages[1001]
+
+    # Load PodNet configuration
+    status, config_data, msg = load_pod_config(config_file)
+    if not status:
+        return False, msg
+
+    enabled = config_data['processed']['enabled']
+    disabled = config_data['processed']['disabled']
+
+    def run_podnet(podnet_node, prefix, successful_payloads):
+        rcc = SSHCommsWrapper(comms_ssh, podnet_node, 'robot')
+        fmt = PodnetErrorFormatter(
+            config_file,
+            podnet_node,
+            podnet_node == enabled,
+            {'payload_message': 'STDOUT', 'payload_error': 'STDERR'},
+            successful_payloads,
+        )
+
+        # Define all payloads
+        payloads = {
+            'check_routes': f'ip netns exec {namespace} ip route show | grep "dev xfrm{vpn_id}" 2>/dev/null || echo "Interface does not exist"',  
+            'add_route': 'ip netns exec {namespace} ip route add %(network)s dev xfrm{vpn_id}',
+            'del_route': 'ip netns exec {namespace} ip route del %(route)s',
+        }
+
+        # Remove existing routes first
+        check_routes_ret = rcc.run(payloads['check_routes'])
+        
+        if check_routes_ret["channel_code"] != CHANNEL_SUCCESS:
+            return False, fmt.channel_error(check_routes_ret, f"{prefix+3}: " + messages[prefix + 3]), fmt.successful_payloads
+        
+        if check_routes_ret["payload_code"] != SUCCESS_CODE:
+            return False, fmt.payload_error(check_routes_ret, f"{prefix+4}: " + messages[prefix + 4]), fmt.successful_payloads
+            
+        # Remove existing routes if any
+        if "Interface does not exist" not in check_routes_ret["payload_message"]:
+            routes = []
+            for line in check_routes_ret["payload_message"].strip().split('\n'):
+                if line and "dev xfrm" in line:
+                    parts = line.split()
+                    if parts and parts[0]:
+                        routes.append(parts[0])
+            
+            for route in routes:
+                payload = payloads['del_route'] % {'route': route}
+                ret = rcc.run(payload)
+                
+        fmt.add_successful('remove_existing_routes', {})
```

Comment:
```suggestion
                fmt.add_successful(f'remove_existing_route_{route}', {})
```

Indenting to be included in the for loop and record each route removed